br/pkg/lightning/mydump/loader_test.go

```diff
@@ -1129,9 +1128,6 @@ func TestSampleParquetDataSize(t *testing.T) {
 	pwriter.RowGroupSize = 128 * 1024 * 1024 //128M
 	pwriter.PageSize = 8 * 1024              //8K
 	pwriter.CompressionType = parquet.CompressionCodec_SNAPPY
-	seed := time.Now().Unix()
-	t.Logf("seed: %d", seed)
```

Comment:
We need to print the current seed, so when the unit tests fails we can set to that specific seed and reproduce it. Can you keep the printing here? like

```
t.Logf("seed: %d. To reproduce the random behaviour, manually set `rand.Seed(seed)`", seed)
```